### PR TITLE
Journal pagination

### PR DESCRIPTION
**Pull request by AJIex1s**

### CODE REVIEW

File: SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml [hunk 1]
```diff
@@ -7,18 +7,18 @@
...
+            <th class="сalling-lessons-for-deleting-groups" colspan="@(Model.Lessons.Items.Count()+1)">
                 <label>Удалить все колонки и даты</label>
                 <i onclick="deleteGroupLessons()" class="fa fa-trash-o delete-group-lessons" title="Удалить все оценки и даты!"></i>
             </th>
             <th class="hidden-header">
             </th>
         </tr>
         <tr>
-            <th>
+            <th class="creating-a-student">
```
Comment (AJIex1s) [inline SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml:21]: class it is not an action. it should be - create-student-controls or smth. similar

File: SchoolJournal.Web/wwwroot/css/site.css [hunk 2]
```diff
@@ -338,4 +338,12 @@ table {
 
 .active-link {
     color: #7CB0D3;
+}
+
+.groups-dropdownlist {
+    text-align: center;
+}
+
+.сalling-lessons-for-deleting-groups {
```
Comment (AJIex1s) [inline SchoolJournal.Web/wwwroot/css/site.css:347]: Rename!

File: SchoolJournal.Web/wwwroot/css/students.css [hunk 3]
```diff
@@ -206,3 +206,7 @@
     width: 43px;
 }
 
+.creating-a-student {
```
Comment (AJIex1s) [inline SchoolJournal.Web/wwwroot/css/students.css:209]: rename

File: SchoolJournal.Web/Views/Account/LoginPage.cshtml [hunk 4]
```diff
@@ -19,14 +19,14 @@
 
         @Html.ValidationSummary(true, "", new { @class = "text-danger" })
         <div class="сol" align="center">
-            <div class="col" align="center">
+            <div class="login-input" align="center">
```
Comment (AJIex1s) [inline SchoolJournal.Web/Views/Account/LoginPage.cshtml:22]: Div not equals Input

File: SchoolJournalCore.Core/QueryableExtensions.cs [hunk 5]
```diff
@@ -24,8 +24,14 @@ public static class QueryableExtensions
             if (totalItems != 0)
             {
                 var lastPageNumber = (int)Math.Ceiling((double)totalItems / paginationParameters.Size);
+
                 int skip = paginationParameters.Skip(lastPageNumber);
 
+                if (totalItems < skip)
```
Comment (AJIex1s) [inline SchoolJournalCore.Core/QueryableExtensions.cs:30]: WTF!?

File: SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml [hunk 6]
```diff
@@ -7,18 +7,18 @@
 <div class="table-responsive journal-table-wrapper">
     <table class="table journal-table table-dark">
         <tr>
-            <th>
+            <th class="groups-dropdownlist">
                 @Html.DropDownList("groups", new SelectList(Model.Groups, "Id", "Name", Model.CurrentGroupId), new { @onchange = "groupChanged(this.value)", @class = "selectpicker btn-lg dropdown-toggle" })
             </th>
-            <th colspan="@(Model.Lessons.Items.Count()+1)">
+            <th class="delete-group-lessons-container" colspan="@(Model.Lessons.Items.Count()+1)">
```
Comment (AJIex1s) [inline SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml:13]: delete-group-lessons-cell

File: SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml [hunk 7]
```diff
@@ -7,18 +7,18 @@
...
                 <label>Удалить все колонки и даты</label>
                 <i onclick="deleteGroupLessons()" class="fa fa-trash-o delete-group-lessons" title="Удалить все оценки и даты!"></i>
             </th>
             <th class="hidden-header">
             </th>
         </tr>
         <tr>
-            <th>
+            <th class="add-student-container">
```
Comment (AJIex1s) [inline SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml:21]: cell

File: SchoolJournal.Web/wwwroot/css/students.css (hunk 3 above)
Comment (Mallusca) [reply SchoolJournal.Web/wwwroot/css/students.css]: done

File: SchoolJournal.Web/wwwroot/css/site.css [hunk 8]
```diff
@@ -338,4 +338,18 @@ table {
...
+.groups-dropdownlist {
+    text-align: center;
+}
+
+.delete-group-lessons-container {
+    text-align: center;
+}
+.login-container {
+    width: fit-content;
+}
```
Comment (AJIex1s) [inline SchoolJournal.Web/wwwroot/css/site.css:352]: formatting "new line"

File: SchoolJournal.Web/wwwroot/css/site.css (hunk 2 above)
Comment (Mallusca) [reply SchoolJournal.Web/wwwroot/css/site.css]: done

File: SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml (hunk 1 above)
Comment (Mallusca) [reply SchoolJournal.Web/Views/Admin/Journal/JournalGrid.cshtml]: done

File: SchoolJournal.Web/Views/Account/LoginPage.cshtml (hunk 4 above)
Comment (Mallusca) [reply SchoolJournal.Web/Views/Account/LoginPage.cshtml]: done

**Pull request merged**